fili-generic-example/src/main/java/com/yahoo/wiki/webservice/data/config/table/GenericTableLoader.java

```diff
@@ -100,7 +104,7 @@ private void configureTables(
                     getPhysicalTableDefinitions(
                             dataSourceConfiguration,
                             dataSourceConfiguration.getValidTimeGrain(),
-                            genericDimensionConfigs.getDimensionConfigs(dataSourceConfiguration)
+                            genericDimensionConfigs.getDimensionConfigs(dataSourceConfiguration.getName())
```

Comment:
We've deprecated `getDimensionConfigs(DataSourceConfiguration dataSourceConfiguration)` mentioned in previous comments - `getDimensionConfigs ` is not taking a whole object, but only taking the part of object that this method really needs.